style.css

```diff
@@ -0,0 +1,229 @@
+body {
+  background-color: white;
+  font-family: "Merriweather", serif;
+}
+
+main {
+  padding-bottom: 100px;
+  padding-left: 50px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+}
+
+header {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  background-color: white;
+  border-bottom: 2px solid #e4e4e4;
+}
+
+h1 {
+  display: block;
+  text-align: center;
+  padding-top: 70px;
+  font-size: 32px;
+}
+
+h2 {
```

Comment:
not a huge deal, but i would avoid styling elements like this. better to add a class. you will see why when we have a larger app 🙂